.github/workflows/dev-workflow.yml

```
@@ -165,12 +165,72 @@ jobs:
         run: |
           npm i -g aws-cdk
           make deploy
+    
+  fetcher:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)